orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/cluster/FindImageFromClusterTask.groovy

```diff
@@ -80,6 +81,7 @@ class FindImageFromClusterTask extends AbstractCloudProviderAwareTask implements
   @Canonical
   static class FindImageConfiguration {
     String cluster
```

Comment:
Can we drop cluster? Or are you keeping it around to avoid breaking things for now?

Reply:
Yeah, I left that in there to avoid breaking things. This is going to be a pattern throughout the whole process of migrating to Moniker. Most stages/tasks classes already have some subset of the fields that are in Moniker (app, stack, detail, cluster). We should probably have a larger discussion about how we want to handle those in the future. For now I am viewing that optimization as a separate task from the migration.